### PR TITLE
Standardizing module import method for pySHiELD

### PR DESCRIPTION
**Pull request by fmalatino**
**Description**
Adding __init__ files to standardize import method of modules

**How Has This Been Tested?**
Tested via lint and currently implemented translate tests
OS: RHEL 8.9 Ootpa
Built conda environment using workflow in github action configurations

**Checklist:**
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules
- [x] New check tests, if applicable, are included
- [x] Targeted model, if this change was triggered by a model need/shortcoming

### CODE REVIEW

**Review by xyuan — DISMISSED** [review]
this looks good to me too,

File: tests/savepoint/translate/translate_gfs_physics_driver.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import copy
 
 import ndsl.dsl.gt4py_utils as utils
-import ndsl.util as util
```
Comment (FlorianDeconinck) [inline tests/savepoint/translate/translate_gfs_physics_driver.py:4]: Renamed that `util` in `init` or something, that's a mistake left from refactor

File: pySHiELD/update/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from .fv_update_phys import ApplyPhysicsToDycore
```
Comment (FlorianDeconinck) [inline pySHiELD/update/__init__.py:1]: @oelbert same comment than for `pyFV3` on small docstring

**Review by FlorianDeconinck — COMMENTED** [review]
Nothing on `stencils` ?

File: pySHiELD/update/__init__.py (hunk 2 above)
Comment (oelbert) [reply pySHiELD/update/__init__.py:1]: For these it's pretty straightforward so definitely think that's a good idea here.

**Comment by FlorianDeconinck** [comment]
Please merge in `develop` instead of `main`

**Comment by oelbert** [comment]
> Nothing on `stencils` ?

I want to rework how the physics modules are structured, get rid of separate stencils and functions directories, have the code in a better-designed structure, like pyFV3, so I told Frank he could skip Stencils for now

**Comment by fmalatino** [comment]
> > Nothing on `stencils` ?
> 
> I want to rework how the physics modules are structured, get rid of separate stencils and functions directories, have the code in a better-designed structure, like pyFV3, so I told Frank he could skip Stencils for now

I am working on them now.

**Review by FlorianDeconinck — DISMISSED** [review]
LGTM, nice cleanup.

File: pySHiELD/stencils/__init__.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+from .microphysics import Microphysics, MicrophysicsState
+
+
+"""
+Microphysics: Class containing methods for updating microphysics and interactions
+MicrophysicsState: Class which initializes defines physics variable types
```
Comment (oelbert) [inline pySHiELD/stencils/__init__.py]:
```suggestion
Microphysics: GFS Cloud Microphysics class
MicrophysicsState: Class containing the state for the GFS Cloud Microphysics
```

File: pySHiELD/__init__.py [hunk 4]
```diff
@@ -1,8 +1,14 @@
...
 
 
 __all__ = list(key for key in locals().keys() if not key.startswith("_"))
 __version__ = "0.2.0"
+
+"""
+PHYSICS_PACKAGES: Enum of available physics packages
+PhysicsConfig: Dataclass containing physics variables as defined by configuration yaml
+PhysicsState: Dataclass which initializes defines physics variable types
+Physics: Class containing methods for updating physics and interactions
```
Comment (oelbert) [inline pySHiELD/__init__.py]:
```suggestion
PhysicsConfig: Class containing physics variables and model configurations
PhysicsState: Class containing the state for the physics schemes
Physics: Class that specifies and executes the subgrid physics schemes
```

File: pySHiELD/update/__init__.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+"""
+ApplyPhysicsToDycore: Class containing methods to apply physics to FV3
+                      discretiztion and prognostic variables
+DycoreToPhysics: Class containing methods for transferring dynamical
+                 core state information to physics state
+UpdateAtmosphereState: Class containing methods for applying tendencies
+                       and keeping prognostic state consistent
+AGrid2DGridPhysics: Class containing methods to update D-grid wind
+                    physics from A-grid state
```
Comment (oelbert) [inline pySHiELD/update/__init__.py]:
```suggestion
ApplyPhysicsToDycore: Class to update the pyFV3 dynamical core state from the physics state
DycoreToPhysics: Class to set the physics state from the pyFV3 dynamical core state
UpdateAtmosphereState: Class containing methods to apply tendencies while keeping prognostic state consistent
AGrid2DGridPhysics: Class to transform wind tendencies from the A-grid to the D grid
```

**Review by oelbert — DISMISSED** [review]
Some suggestions for the docstrings but looks good to me

**Review by oelbert — DISMISSED** [review]
👍

**Review by oelbert — APPROVED** [review]
👍

**Pull request merged**